jekyll/_cci2_ja/nomad.adoc

```diff
@@ -22,13 +22,13 @@ toc::[]
 .Nomad クラスタの管理
 image::nomad-diagram-v2.png[Diagram of the Nomad cluster]
 <<<
-- **Nomad サーバー:** Nomad サーバーは、Nomad クラスタの "頭脳" です。 ジョブを受け取り、Nomad クライアントに割り当てます。 CircleCI では、Nomad サーバーは Docker コンテナとしてサービスマシンで実行されます。
+- *Nomad サーバー:* Nomad サーバーは、Nomad クラスタの "頭脳" です。 ジョブを受け取り、Nomad クライアントに割り当てます。 CircleCI では、Nomad サーバーは Docker コンテナとしてサービスマシンで実行されます。
 
-- **Nomad クライアント:** Nomad クライアントは、Nomad サーバーによって割り当てられたジョブを実行します。 通常、Nomad クライアントは専用のマシン (多くの場合は VM) 上で実行されるため、マシンの能力を最大限に活用できます。 複数の Nomad クライアントで 1つのクラスタを構成することができ、Nomad サーバーはスケジュールアルゴリズムに従ってクラスタにジョブを割り当てます。
+- *Nomad クライアント:* Nomad クライアントは、Nomad サーバーによって割り当てられたジョブを実行します。 通常、Nomad クライアントは専用のマシン (多くの場合は VM) 上で実行されるため、マシンの能力を最大限に活用できます。 複数の Nomad クライアントで 1つのクラスタを構成することができ、Nomad サーバーはスケジュールアルゴリズムに従ってクラスタにジョブを割り当てます。
 
-- **Nomad ジョブ:** Nomad ジョブは、ユーザーによって提供される仕様で、Nomad のワークロードを宣言します。 Nomad ジョブは、CircleCI ジョブの実行に対応しています。 たとえば 10個の並列実行を使用するジョブの場合、Nomad は 10個のジョブを実行します。
+- *Nomad ジョブ:* Nomad ジョブは、ユーザーによって提供される仕様で、Nomad のワークロードを宣言します。 Nomad ジョブは、CircleCI ジョブの実行に対応しています。 たとえば 10個の並列実行を使用するジョブの場合、Nomad は 10個のジョブを実行します。
```

Comment:
This is also similar to above the bold is removed. I recommend to use two asterisks.